README.md

```diff
@@ -68,10 +66,6 @@ default location for staging temporary files to the /tmp directory.
 The normal default is /shm. /shm is a RAM disk which defaults to a 64 MB size
 in Docker containers, too small for typical scientific computing.
 
-# Install a new PyPi Package Dependency
-1. Add new dependency to `requirements-core.txt` file. Include the version
-2. Run script `generate-requirements-full.sh`
```

Comment:
Dropping this bit to discourage adding new packages in general. The more packages we consider as "core", the harder it is for us to maintain a resolvable lock / requirements-full.txt.